src/pushNotification.ts

```diff
@@ -45,7 +45,7 @@ const walletServiceClient = {
   async pushRegister(wallet: HathorWalletServiceWallet, payload: PushRegisterRequestData): Promise<PushRegisterResponseData> {
     const axios = await axiosInstance(wallet, true);
 
-    const response = await axios.post<PushRegisterResponseData>('push/register', payload);
+    const response = await axios.post<PushRegisterResponseData>('wallet/push/register', payload);
```

Comment:
for the WS no, but meanwhile this PR I changed the lambda path and didn't update the lib. Can we use this PR to convey this change?

Reply:
It's ok because it's a small change but you should update the acceptance criteria to reflect this, otherwise we may think this is wrong or a leftover from a previous merge